test/integration/custom-server/test/index.test.js

```diff
@@ -101,17 +101,15 @@ describe.each([
     })
 
     it('should set the assetPrefix to a given request', async () => {
-      for (let lc = 0; lc < 1000; lc++) {
```

Comment:
Was it intended to be a stress test?

Reply:
Yeah, I think you're right. How about reducing the number of iterations? 2000 renders seem too much for validating assetPrefix on different requests.